src/components/Calculator.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable react/destructuring-assignment */
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+import './calculator.css';
+
+const Calculator = () => {
+  const [input, setInput] = useState({
+    total: 0,
+    next: null,
+    operator: null,
+  });
+  const handleClick = (e) => {
+    const btnClick = e.target.innerText;
+    const response = calculate(input, btnClick);
+    setInput(response);
```

Comment:
- [ ] As requested by the previous reviewer, Please change this component to a class-based component to follow the project requirements.
![image](https://user-images.githubusercontent.com/83803414/183135624-a7f49e74-dee5-4de6-91fb-5eb6968b33d4.png)